### DISCUSSION TITLE
I am getting an error saying "Error: unexpected end of stream"

### DISCUSSION THREAD

**Discussion by arjunvnd** [callstack/repack#758] [Q&A]
When trying to connect to a microfrontend repo from the host app, I am getting an error 

`[ScriptManager] Failed to load script: [ScriptCachingFailure] {"cache": false, "caller": undefined, "locator": {"absolute": false, "body": undefined, "fetch": true, "headers": undefined, "method": "GET", "query": "platform=android", "timeout": 30000, "url": "URL to repo", "verifyScriptSignature": "off"}, "scriptId": "RepoName"} {"originalError": [Error: unexpected end of stream]}`

I have tried doing the adb reverse command as suggested in one of the issues previously raised, but the error seems to be different from the one I am getting.

Is there a place in the documentation where I can see the possible find the errors and their work arounds. 
OS - windows

Any help will be really appreciated.

**Comment by jbroma** [comment]
hi @arjunvnd, can you open an issue and provide a reproduction repository?
